active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,618 @@
+```CS 
+public void SetTitleBar(UIElement); 
+```
+
+#### Parameters
+UIElement: Custom XAML content that should act as the title bar. To use multiple objects, wrap them in a container element such as one derived from Panel.
+
+#### Remarks
+
+- **Win32**: This method do nothing when running in Win32.
+- **UWP**: Continue using the OS behavior. More info [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.window.settitlebar) 
+
+### Window.Title property **[NEW]**
+
+Set or get a string to display on the Title. 
+
+```CS
+public string Title { get; set; }
+```
+#### Sample
+```XML
+```XML
+<Window 
+    x:Class="MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    Title="Main Window" 
+>
+    
+</Window>
+```
+
+#### Remarks
+This is not a DependencyProperty.
+
+- **UWP**: This string is not displayed in the Window's Title bar when SetTitleBar method is used or set a not null content. The Title is still used in Alt+Tab scenarios,
+
+
+### Window.Current property
+
+Gets the window of the current thread.
+
+```CS
+public static Window Current { get; }
+```
+
+#### Remarks
+- **UWP**: Gets the Window object for the current thread. This is based on the asumption that UWP only has just one CoreWindow.
+- **Win32**: Returns null when running in Win32 given there could be multiple Windows on the same thread. 
+
+### **Window lifetime management**
+
+### Window.Activate method 
+Attempts to activate the window by making it visible, bringing it to the foreground and setting the input focus to it. If the Window can’t be activated, it fails silently.
+
+```CS
+public void Activate()
+```
+
+### Window.Activated Event
+ Occurs when the window has successfully been activated or deactivated. If the focus takes away from the Window, the deactivate event will occur. If the focus returns to the window the activate event occurs.
+```CS
+public event WindowActivatedEventHandler Activated
+```
+
+#### Samples
+
+```CS
+window.Actived += OnActivated;
+```	
+
+```CS
+void OnActivated(object sender, WindowActivatedEventArgs e)
+{
+   if (e.WindowActivationState == WindowActivationState.CodeActivated)
+    {
+        // 
+    }
+    else if (e.WindowActivationState == WindowActivationState.Deactivated)
+    {
+        // 
+    }
+    else if(e.WindowActivationState == WindowActivationState.PointerActivated)
+    {
+       //
+    }
+}
+```
+#### Remarks
+- **UWP**: Current behavior is [here](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.window.activated)
+
+### Window.Close method
+
+Closes and destroy the window. The Closed event occurs when a Window closes. 
+```CS
+public void Close()
+```
+
+#### Remarks
+- **UWP**: UWP app typically use a single Window instance, and do not open and close additional Window instances. Additionally, you will not typically provide custom UI to enable users to close the app window.
+
+### Window.Closed event
```

Comment:
>Closed [](start = 11, length = 6)

Do you want to have typical functionality here:
1. Allow app to get a deferral and start saving state.
2. Potentially override the "closing" behavior.

I'd recommend talking with Roberth and Francis on real-world scenarios here.

Reply:
Thanks for the piece of advice!  We will talk to them. Resolving this conversation,